metrics/grafana/ticdc.json

```diff
@@ -130,8 +130,8 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 1,
-  "id": null,
-  "iteration": 1667966386939,
+  "id": 27,
```

Comment:
What does this id stand for, is it conflict with `"id":27` in L2348

Reply:
I don't know what it means. But I think they are at different levels.